notebooks/community/vertex_endpoints/optimized_tensorflow_runtime/bert_optimized_online_prediction.ipynb

```diff
@@ -1510,7 +1510,7 @@
       "source": [
```

Comment:
<p>Throughout notebook, replace:</p><p> </p><p>we -&gt; you</p><p>our -&gt; your</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/630/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>used wording from a new notebook_template.ipynb and updated where appropriate</p>